pexpect/pxssh.py

```diff
@@ -527,11 +528,14 @@ def set_unique_prompt(self):
         self.sendline("unset PROMPT_COMMAND")
         self.sendline(self.PROMPT_SET_SH) # sh-style
         i = self.expect ([TIMEOUT, self.PROMPT], timeout=10)
-        if i == 0: # csh-style
-            self.sendline(self.PROMPT_SET_CSH)
+        if i == 0: # zsh-style
+            self.sendline(self.PROMPT_SET_ZSH)
```

Comment:
I'm somewhat agnostic about the order of trying `zsh` vs `csh` here; with MacOS moving to `zsh` as the default it seems like that's likely to be more common these days.

Reply:
I'd prefer if this stayed in the original order (add zsh after csh) for compatibility reasons for other platforms and to not disturb an already established order.

Reply:
Done